exonum-java-binding-cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/transactions/CreateWalletTxTest.java

```diff
@@ -60,7 +60,7 @@ public void walletIsValidWithCorrectOwnerKey() {
   public void constructorRejectsInvalidSizedKey() {
     PublicKey publicKey = PublicKey.fromBytes(new byte[1]);
 
-    expectedException.expectMessage("Public key must have correct size");
+    expectedException.expectMessage("Public key has invalid size (1), must be 32 bytes long.");
```

Comment:
`must be " + CRYPTO_SIGN_ED25519_PUBLICKEYBYTES + " bytes long`?

Reply:
Though it looks ugly, so I don't insist.

Reply:
Yes, would be rather long. I'd keep the message, if we ever update the cryptosystem, will have a failing test.